src/shared/vk.h

```diff
@@ -14,6 +14,7 @@ namespace dxvk {
         virtual ~Vk();
 
         [[nodiscard]] virtual bool IsAvailable() const;
+        [[nodiscard]] virtual PFN_vkVoidFunction GetInstanceProcAddr(VkInstance vkInstance, const char* name) const;
```

Comment:
Does this make private `GetInstanceProcAddress` below redundant or do you still want to keep it because it's a template?

Reply:
Yes. I wanted to keep runtime changes to a minimum and I indeed still like the more beautiful version for private use.